code/datums/diseases/legionvirus.dm

```diff
@@ -1,28 +1,29 @@
-/datum/disease/transformation/legionvirus
-	name = "Soulus Saturation"
+/datum/disease/transformation/legionvirus //Diseases are a quick way to exposit a bunch of information onto players, most of the effects here are handled by the legion skull organ from /mob/living/simple_animal/hostile/mining_mobs/hivelord.dm
+	name = "Legion Infection"
 	max_stages = 5
 	spread_flags = DISEASE_SPREAD_NON_CONTAGIOUS
-	cure_text = "System Cleaner"
-	cures = list(/datum/reagent/medicine/system_cleaner)
-	agent = "Soulus Dust"
+	cure_text = "Surgically removing the skull from the patient's chest; applications of spaceacillin or synaptizine can delay onset of the infection."
+	agent = "Legion skull"
 	viable_mobtypes = list(/mob/living/carbon/human)
 	visibility_flags = 0
-	stage_prob = 5
-	cure_chance = 20
-	desc = "If left untreated, this disease will cause the infected to transform into a legion."
+	stage_prob = 0 // WOOOOOO SNOWFLAKE!!!!!!! WOOOOOO!!!!
+	desc = "If left untreated, the skull will slowly overtake its host's body, eventually growing into a legion."
 	severity = DISEASE_SEVERITY_HARMFUL
-	disease_flags = CURABLE
+	disease_flags = NONE
+	visibility_flags = HIDDEN_PANDEMIC
 	bypasses_immunity = TRUE
 
-	stage1 = list("Your joints itch.")
+	stage1 = list("You feel a dull pain in your chest.")
 	stage2 = list("Your head begins to ache.")
-	stage3 = list("Ash begins to flake off your skin.")
-	stage4 = list("<span class='warning'>You feel like your head is splitting in two!</span>")
-	stage5 = list("<span class='warning'>You feel something growing inside your chest!</span>")
+	stage3 = list("Something moves underneath your skin.")
+	stage4 = list("<span class='warning'>You feel something pressing against your skin!</span>")
+	stage5 = list("<span class='warning'>Your skin begins to tear apart-!</span>")
```

Comment:
span defines these
notice on the ones that arent warning

Reply:
Handled